hdl/modules/fofb_sys_id/cheby/wb_fofb_sys_id_regs.cheby

```diff
@@ -7,24 +7,160 @@ memory-map:
   children:
     - block:
         name: bpm_pos_flatenizer
-        description: Interface to bpm_pos_flatenizer regs
+        description: Interface to BPM positions flatenizers regs
         address: 0x00000000
         children:
+          - reg:
+              name: ctl
+              width: 32
+              access: rw
+              address: next
+              description: |
+                BPM positions flatenizers control register
+              children:
+                - field:
+                    name: base_bpm_id
+                    range: 7-0
+                    description: |
+                      Together with max_num_cte, defines the ranges of BPM
+                      positions indexes being flatenized, which are given by
+
+                      [base_bpm_id, base_bpm_id + max_num_cte): BPM x positions
+                      [base_bpm_id + 256, base_bpm_id + 256 + max_num_cte): BPM y positions
           - reg:
               name: max_num_cte
               width: 16
               access: ro
               address: next
               description: |
                 Maximum number of BPM positions that can be flatenized per axis
+                (x or y)
+    - block:
+        name: prbs
+        description: Interface to PRBS-related regs
+        address: next
+        children:
           - reg:
-              name: base_bpm_id
-              width: 8
+              name: ctl
+              width: 32
               access: rw
               address: next
               description: |
-                Together with max_num_cte, defines the range of BPM position
-                indexes being flatenized, which is given by
-                [base_bpm_id, base_bpm_id + max_num_cte) -> BPM x positions; and
-                [base_bpm_id + 256, base_bpm_id + 256 + max_num_cte) -> BPM y
-                positions. The valid range of this register is [0, 255].
+                PRBS distortion control register
+              children:
+                - field:
+                    name: rst
+                    x-hdl:
+                      type: autoclear
+                    range: 0
+                    description: |
+                      Resets PRBS
+
+                      write 0: no effect
+                      write 1: resets PRBS (this bit autoclears)
```

Comment:
This has to be updated: "enables PRBS reset via trigger", and it doesn't autoclear.

Should we document the logical trigger (probably from the SYSID trigger mux?) responsible for this here? Or somewhere else?

Reply:
Not here, the way you connect the trigger is defined in the top level RTL entity.

Reply:
@ericonr , I'll add a new commit with trigger support.